### DISCUSSION TITLE
PyBullet VR with Oculus quest 2 (+ Oculus Link or Oculus Air Link)

### DISCUSSION THREAD

**Discussion by asaka-t** [bulletphysics/bullet3#3851] [Q&A] [answered]
I confirmed that PyBullet can be used in VR with Oculus Rift, but App_PhysicsServer_SharedMemory_VR_vs2010_x64_release.exe did not work properly with Oculus quest 2 (+ Oculus Link or Oculus Air Link).
If this works, you'll be able to use PyBullet in VR without a cable.
Please let me know if this is possible. Also, if the current version does not support it, could you update it?

**Comment by asaka-t** [comment, marked as answer]
I'm sorry, but I confirmed that PyBullet can also be used in VR for Oculus Quest 2 (+ Oculus Quest 2 or Oculus Air Link).
The cause is hardware, and in the case of a multi-GPU configuration, it seems that it did not work properly because the GPU used for rendering could not be properly determined. By removing the GPU, it worked correctly.

**Reply by aha85b** [reply]
how did you connect it, do you have a tutorial for that?
